contracts/libraries/Message.sol

```diff
@@ -81,7 +81,9 @@ library Message {
 
     function hashMessage(bytes message) internal pure returns (bytes32) {
         bytes memory prefix = "\x19Ethereum Signed Message:\n";
```

Comment:
check that the message length is really 116.

Reply:
it's already been done in all places before that call is made.

Reply:
thanks!